src/program/lwaftr/tests/selftest.sh

```diff
@@ -6,18 +6,18 @@
 export TESTS_DIR=`dirname "$0"`
 export PYTHONPATH=${TESTS_DIR}
 
-# Only run tests in the passed subdirectory of $TESTS_DIR.
+# Only run tests in the chosen test file (without the _test.py suffix).
 if [[ -n $1 ]]; then
```

Comment:
Need quotes around $1, I think; otherwise `foo.sh "a b"` expands out to `[[ -n a b ]]` which AFAIU is bad syntax.